appium-dotnet-driver/Appium/Tizen/TizenDriver.cs

```diff
@@ -64,24 +86,88 @@ public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities)
         {
         }
 
-        // TODO: This feature may not work properly at the moment.
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified Appium local service and desired capabilities
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities of the browser.</param>
         public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities)
             : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform))
         {
         }
 
-        // TODO: This feature may not work properly at the moment.
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified remote address, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="remoteAddress">URI containing the address of the WebDriver remote server (e.g. http://127.0.0.1:4723/wd/hub).</param>
+        /// <param name="desiredCapabilities">An <see cref="DesiredCapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
         public TizenDriver(Uri remoteAddress, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
             : base(remoteAddress, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
         {
         }
 
-        // TODO: This feature may not work properly at the moment.
+        /// <summary>
+        /// Initializes a new instance of the TizenDriver class using the specified Appium local service, desired capabilities, and command timeout.
+        /// </summary>
+        /// <param name="service">the specified Appium local service</param>
+        /// <param name="desiredCapabilities">An <see cref="ICapabilities"/> object containing the desired capabilities.</param>
+        /// <param name="commandTimeout">The maximum amount of time to wait for each command.</param>
         public TizenDriver(AppiumLocalService service, DesiredCapabilities desiredCapabilities, TimeSpan commandTimeout)
             : base(service, SetPlatformToCapabilities(desiredCapabilities, Platform), commandTimeout)
         {
         }
 
+        public bool CompareImages(Image firstImage, Image secondImage)
+        {
+            if (firstImage == null || secondImage == null)
+            {
+                throw new NullReferenceException("Screenshot and Rectangle should not be null");
+            }
+
+            var firstBitmap = new Bitmap(firstImage);
+            var secondBitmap = new Bitmap(secondImage);
+            Color firstPixel;
+            Color secondPixel;
+            int firstGrayScale = 0;
+            int secondGrayScale = 0;
+
+            for (int i = 0; i < firstImage.Width; i++)
```

Comment:
where comes this approach?  just of quick scan, it seems it has poor performance.

could you optiming it? 

For example, we can check length/size of both images before looping. Or we can consider to use `LockBitmap` ,.. and so on.

Reply:
Performance comparison between LockBits and GetPixel.

https://mfranc.com/programming/operacje-na-bitmapkach-net-1/

Reply:
Done